frontend/vite.config.ts

```diff
@@ -24,5 +24,14 @@ export default defineConfig({
       }
     }),
     tsconfigPaths()
-  ]
+  ],
+  resolve: {
+    alias: {
+      crypto: 'crypto-browserify'
+    }
+  },
+  build: {
+    sourcemap: 'inline',
```

Comment:
To avoid increasing size of bundled JS file (it'll use source map as needed).
```suggestion
    sourcemap: true,
```

https://developers.cloudflare.com/pages/functions/source-maps/